### PR TITLE
[#5404] apps/organisations: make organisation slugs editable in django-form

### PR DESCRIPTION
**Pull request by sabinammm**
[](https://taiga.liqd.net/)#5404
[mB] make Organisation and project URLs changeable via django

### CODE REVIEW

File: meinberlin/apps/organisations/forms.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Meta:
         }
 
     def clean(self):
+        self._validate_unique = True
```
Comment (fuzzylogic2000) [inline meinberlin/apps/organisations/forms.py:27]: @sabinammm That was hard to find. Here in mB we use this form in the django-admin to validate the groups. In the clean-method of django, the cleaned_data is returned as well, but first, it's validated that the fields are unique. https://github.com/django/django/blob/7c318a8bdd66f8c5241864c9970dddb525d0ca4c/django/forms/models.py#L432 I added that to our method overwriting the clean-method as well.

File: meinberlin/apps/organisations/models.py [hunk 2]
```diff
@@ -9,7 +9,14 @@
 
 
 class Organisation(models.Model):
-    slug = AutoSlugField(populate_from='name', unique=True)
+    slug = AutoSlugField(
+        populate_from='name',
+        unique=True,
+        editable=True,
+        help_text=_(
```
Comment (fuzzylogic2000) [inline meinberlin/apps/organisations/models.py:16]: So, now you can remove the help text again and redo the migration. :)

**Review by fuzzylogic2000 — COMMENTED** [review]
Could you remove the second commit, so that we can do that with the update of a4 in a separate PR?

File: meinberlin/apps/organisations/models.py (hunk 2 above)
Comment (sabinammm) [reply meinberlin/apps/organisations/models.py:16]: Awesome:) Thank you! I will do this and remove the second commt and put it into a different PR

File: meinberlin/apps/organisations/forms.py (hunk 1 above)
Comment (sabinammm) [reply meinberlin/apps/organisations/forms.py:27]: :pray:

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

**Pull request merged**